src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -674,6 +674,13 @@ def load_arguments(self, _):
                    "--ignore-pod-disruption-budget", "-i"], action=get_three_state_flag(), is_preview=True,
                    help='delete an AKS nodepool by ignoring PodDisruptionBudget setting')
 
+    with self.argument_context('aks machine') as c:
+        c.argumemt('cluster_name', options_list=['--cluster-name'], help='The cluster name.')
```

Comment:
could omit the `options_list` if the option name is the same as parameter name (cli framework would help replace `_` with `-` and add leading `--`)